pkg/amazon/sdk/sdk.go

```diff
@@ -66,7 +65,8 @@ func (s sdk) CheckRequirements(ctx context.Context) error {
 	}
 	serviceLongArnFormat := settings.Settings[0].Value
 	if *serviceLongArnFormat != "enabled" {
-		return errors.New("this tool requires the \"new ARN resource ID format\"")
+		return fmt.Errorf("this tool requires the \"new ARN resource ID format\"."+
```

Comment:
Might be good to add "Learn more: https://aws.amazon.com/blogs/compute/migrating-your-amazon-ecs-deployment-to-the-new-arn-and-resource-id-format-2/"

Reply:
Yes, the more info we put the better it is confusing.